pages/learn/develop/blocks.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Develop with blocks
+excerpt: A drop-in chunk of functionality that can be added to your application to provide.
+---
+
+# Getting Started with blocks
+
+A software package is a collection of individual files or resources that are packed together to provide a certain functionality as part of a larger system. Blocks are just that. They are drop-in, independent chunks of functionality built to handle a specific task, allowing you to focus on solving the hard problems instead.
+
+A block is a container image that can be added to a multi-container app in order to provide specific functionality. Blocks are intended to reduce the friction of application development by making it easier to re-use functionality, much like software packages. Similar to how software developers don't want to rewrite common functions, this is a DRY approach for IoT developers to not create containers of every function and re-use what's already out there for their application to work.
```

Comment:
"....much like software packages provide functionality to a developer writing code".

Either link to an explanation of "DRY" or remove it. Currently we're tossing in a term people might not know.

I think the sentence beginning "Similar" needs a rewrite. Maybe:

"Similar to how software developers don't want to keep rewriting common functions, IOT developers don't want to keep creating containers for common tasks, such as handling Bluetooth connections or interacting with attached displays. Blocks provide solutions to common IOT application requirements, which developers can re-use to enable their Fleet."

Reply:
Love it, good point on removing DRY. Added complexity with not a lot of impact. Replace the line with your suggestion